pushy-donkeys/workforce-management
# seed file works

Fixes for #17 #16/

Changes proposed in this pull request:
- Modified dbinitialize, and the startup.cs
- made slight changes to a few models
-

How to test:
- pull down my branch
- build/do Add-Migration (if you dont have all of the dbSets ready to go)
- then Update-Database and run the server.
- at the end of the URL type /employees or /computers and so on to view the different links

A KNOWN BUG: THE NAMES OF EMPLOYEES APPEAR 3 TIMES EACH

@PushyDonkey/
